lang/en/texts/contribute-to-open-food-facts-by-playing.html

```diff
@@ -0,0 +1,73 @@
+<!-- no side column -->
+
+<h1>Play and contribute to Open Food Facts!</h1>
+
+<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteer developers.</h2>
+
+<div class="button_div unselectbuttondiv_front_en">
+	<a href="https://hunger.openfoodfacts.org/" class="button">Play now</button></a>
+<div>
+<p>We thank the many volunteers who have already <a href="https://world-en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
+<p><strong>This game allows you to modify products easily, quickly and in a fun way</strong>. Enough to make contributing to an Open Source project addictive.</p>
+<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions of people to choose better foods as well as public health research teams for their studies.</p>
+
+<p class="callout panel">To start playing, you just have to log in to your Open Food Facts account and then click on <a href="https://hunger.openfoodfacts.org/">this link</a>.</p>
+
+<p><strong>Want to know more before you get started?</strong>Find out more about the game below!</p>
+
+<h3>How does this game work?</h3>
+
+<p>The game randomly loads a product from Open Food Facts that needs to be completed. The interface is simple, allowing you to interact by answering a question about an attribute of the product by "Yes, Skip or No" and get more product information if needed.</p>
+
+<p>The interface is divided into 3 parts:</p>
+<div><img style="max-height:400px" src="/images/misc/hunger-game-0.png"></div><br>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
+
+<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of questions you wish to answer:</p>
+<ul>
+<li><strong>label</strong>: Questions relating to the "label" field (eg. certification, product origin, etc.)</li>
+<li><strong>category</strong>: Questions relating to the "category" field (eg drinks, sweets, etc.)</li>
+<li><strong>brand</strong>: Questions relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
+<li><strong>product_weight</strong>: Questions relating to the weight of the product (eg 100 g, 1L, etc.)</li>
+</ul>
+<p>Below we find:</p>
+<ol>
+<li>A search field, allowing you to specify which brand, label, category you want to work on,</li>
+<li>The question to be answered in English. In English, this gives: "Does this product correspond to the indicated value?"</li>
```

Comment:
Maybe just:

"The question to be answered".

The "English" part is confusing, let's assume people who will use Hunger Game know some English (otherwise they won't use it anyhow).